golang/cosmos/app/app.go

```diff
@@ -475,7 +477,9 @@ func NewAgoricApp(
 	app.SwingSetSnapshotter = *swingsetkeeper.NewExtensionSnapshotter(
 		bApp,
 		&app.SwingStoreExportsHandler,
-		app.SwingSetKeeper.ExportSwingStore,
+		func(ctx sdk.Context) sdk.Iterator {
+			return app.SwingSetKeeper.GetSwingStore(ctx).Iterator(nil, nil)
+		},
```

Comment:
I think this section would be more readable if the function were given a name.
```suggestion
		getSwingStoreExportDataShadowCopy,
```